db/models/ProjectSchema.js

```diff
@@ -15,6 +15,13 @@ const projectSchema = new Schema(
   {
     title: { type: String, required: true },
     image: { type: String },
+    cover: {
+      type: new Schema({
+        width: Number,
+        height: Number,
+        url: String,
+      }),
```

Comment:
inlining a Schema can be optimised by either just using an object here as a type or moving the Schema into a constant and refering here to it. 
```js
cover: {
      type: {
        width: Number,
        height: Number,
        url: String,
      },
      required: true
   }
```